app/controllers/reports/regions_controller.rb

```diff
@@ -101,6 +101,29 @@ def download
     end
   end
 
+  def who_report
+    authorize { current_admin.accessible_facilities(:view_reports).any? }
+
+    # move to before action or to another controller
+    @period = Period.new(type: params[:period], value: Date.current)
+    unless @period.valid?
+      raise ArgumentError, "Invalid Period #{@period} #{@period.inspect}"
+    end
+    unless @period.month?
+      raise ArgumentError, "Period must be month"
+    end
+
+    csv = WhoReportService.new(@region, @period).report
+    report_date = @period.to_date.strftime("%B-%Y").downcase
```

Comment:
You can use our custom [date/time format](https://github.com/simpledotorg/simple-server/blob/master/config/initializers/time_and_date_formats.rb) for this.

```
  @period.to_s(:month_year).downcase
```